src/spectrecoff/Layout.fs

```diff
@@ -15,25 +15,13 @@ type Padding =
     | HorizontalVertical of int*int
     | TopRightBottomLeft of int*int*int*int
 
-type Style = 
-    | Bold
-    | Dim
-    | Italic
-    | Underline
-    | Invert
-    | Conceal
-    | Slowblink
-    | Rapidblink
-    | StrikeThrough
+open Spectre.Console
+type Look = 
+    { Decoration: Decoration list;
+      Color: Color }
 
-let stringifyStyle style = 
-    match style with
-    | Bold -> "bold"
-    | Dim -> "dim"
-    | Italic -> "italic"
-    | Underline -> "underline"
-    | Invert -> "invert"
-    | Conceal -> "conceal"
-    | Slowblink -> "slowblink"
-    | Rapidblink -> "rapidblink"
-    | StrikeThrough -> "strikethrough"
\ No newline at end of file
+let private aggregate decorations =
+    decorations |>  List.reduce (|||)
```

Comment:
Aha, this black magic can be used because Decoration is a flag enum. Very interesting :)

Reply:
Yes. | mean, obv., duh.